src/Workspaces/SerializationConverters.cs

```diff
@@ -372,7 +372,7 @@ public override void WriteJson(JsonWriter writer, object value, JsonSerializer s
             writer.WriteValue(anno.AnnotationText);
             writer.WritePropertyName("SelectedModels");
```

Comment:
@ramramps this will still write a property called `selectedModels` into the annotation, can you remove this?